lib/render/layout.php

```diff
@@ -289,14 +289,13 @@ function RenderToolbar($user, $permissions = 0)
         echo "<div>";
         echo "<ul>";
         echo "<li><a href='/user/$user'>Profile</a></li>";
-        //echo "<li><a href='/feed.php?i=1'>Feed</a></li>";
-        //echo "<li><a href='/feed.php'>Friends Feed</a></li>";
+        echo "<li><a href='/gameList.php?d=$user'>My Sets</a></li>";
+        echo "<li><a href='/ticketmanager.php?u=admin'>My Tickets</a></li>";
```

Comment:
I believe this should be `u=$user` here instead of `u=admin`

Reply:
Oh, wow, good catch. I'll make the change.